### PR TITLE
Eobs 612 clinical review task triggered

### PR DESCRIPTION
**Pull request by Gimpneek**

### CODE REVIEW

File: nh_eobs_mental_health/models/nh_clinical_patient_observation_ews.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        # Find all ews that have been done since the activity
+        activity_model = self.env['nh.activity']
+        activity = activity_model.browse(activity_id)
+        ews_gen = self.get_child_activity(
+            activity_model, activity, self._name, context=self._context)
+        next_full_ews = list(ews_gen)
+        next_ews_complete = False
+        if next_full_ews:
+            next_ews_complete = next_full_ews[0].state == 'completed'
+        if not next_full_ews and not next_ews_complete:
```
Comment (Gimpneek) [inline nh_eobs_mental_health/models/nh_clinical_patient_observation_ews.py]: Should be an OR

**Comment by moderatemisbehaviour** [comment]
LGTM :shipit: :+1:

**Comment by moderatemisbehaviour** [comment]
:shipit:

**Comment by moderatemisbehaviour** [comment]
:shipit:

**Comment by moderatemisbehaviour** [comment]
LGTM 👍 :shipit:

**Comment by moderatemisbehaviour** [comment]
Looks guurd :shipit: :+1:

**Comment by moderatemisbehaviour** [comment]
coolcoolcool :shipit:

**Comment by moderatemisbehaviour** [comment]
Ooh this is a much nicer solution 😄 satisfying to see all that deleted code  :+1: :shipit:

**Comment by Nayira** [comment]
Test for Story EOBS-612 and Issues EOBS-680, EOBS-681, EOBS-688 and EOBS-692 have passed :shipit:

**Pull request merged**